Crypto/Crypto/Presentation/ViewModels/ForgotPasswordViewModel.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  ForgotPasswordViewModel.swift
+//  Crypto
+//
+//  Created by Orlando Nicolas Marchioli on 03/08/2023.
+//
+
+import Foundation
+import Firebase
+
+class ForgotPasswordViewModel: ObservableObject{
+    @Published var alert: Bool
```

Comment:
@OrlandoNicolasMarchioli, please remind me to return here, we going to implement on Publisher per ViewModel and one state that implements copyWith or cloneWith pattern